tabs/receiver_msp.js

```diff
@@ -14,7 +14,7 @@ var
         aux1: 4,
         aux2: 5,
         aux3: 6,
-        aux4: 7,
+        aux4: 7
```

Comment:
Please, don't mix functional changes with style changes. Also, why remove the trailing comma? It works fine in modern browsers and makes diffs adding elements to the array/object less verbose.

Reply:
And here we come to very nasty problem about configurator.... In general you are right. Although... I'm using IntelliJ Idea to work on JS and such things are treated as errors witch big huge red label. And when I started working on Configurator I've decided that every time I work on the file, I will not commit with validation errors. Reason was simple: every file was marked red

Also, if you think about it, modern browsers do not require semicolon at the on of line, but somehow we add them...

So, while I understand your point, I do not agree ;)

Reply:
I'd say the problem is in your editor configuration then. The only reason we didn't add the trailing comma in JS during the early 2000's was because IE didn't accept that syntax, and that's probably why your editor is marking them as errors. Fortunately all modern browsers (including Chrome since its first version) accept the trailing comma, so we can write cleaner code these days.

Trailing commas are used in virtually any language to declare multiline comma-separate items. Otherwise when you add a new item the diff will be two lines: one to with the new line and another one appending the comma. That's just noise.

Semicolons at EOL have been optional in JS since its inception, but best coding practices have always suggested (or required) to write them explicitly anyway because it makes the code easier to read and maintain. And that's exactly what trailing commas do.

Reply:
Let's just assume that I'm from an old school and trailing coma is a bad thing for me. And not only me, since also http://jslint.com/ does not likes trailing coma, it's not allowed by JSON specification and so on, so let's not start a holy war
I promise not start complaining when I see one and you do not complain when I remove one. Deal?

Reply:
@DzikuVx Seems like a fair deal 👍